lib/util/path_stack.cpp

```diff
@@ -41,8 +44,7 @@
 
 
 struct path_stack_struct {
-  stringlist_type * stack;
-  stringlist_type * storage;
+  std::stack<std::string> stack;
```

Comment:
A very minor comment here. I guess a path stack would hardly have more than 10 or 20 elements. In that case, it is better (performance-wise) to use `std::stack< std::string, std::vector<std::string> >`, or even just `std::vector<std::string>`

Reply:
OK; `std::vector<std::string>` it is.